docs/scenarios_authoring/BasicNodes.md

```diff
@@ -4,77 +4,75 @@ sidebar_position: 2
 
 # Basic Components
 
-Nodes work with a data records. They can produce, fetch, send, collect data or organize data flow. Each node has at least two parameters: `Name` and `Description`. Name has to be unique in a scenario. Description is a narrative of your choice.
+Nodes work with data records. They can produce, fetch, send and collect data or organize the flow. Each node has at least two parameters: `Name` and `Description`. Name has to be unique in a scenario. Description is a narrative of your choice.
 
-Most of the nodes, with source and sink nodes being notable exceptions, have both input and at least one output flow.
+Most of the nodes have both input and at least one output flow. Source and sink nodes are an exception.
 
 Sinks and filters can be disabled by selecting `Disable` checkbox.
 
 &nbsp;
 ## Variable
 
-A Variable component is used to declare a new variable; in the simplest form a variable declaration looks like in the example  below. As the event was read from the Kafka topic, the `#input` variable holds its content and  its value is assigned to a newly declared `myFirstVariable` variable.
+A Variable component is used to declare a new variable. In its simplest form a variable declaration looks like the example  below. As the data record has been read from a data source, the `#input` variable holds the data record's value. After that, the record's (`#input`) value is assigned to a newly declared `myFirstVariable` variable.
 
 ![alt_text](../autoScreenshotChangeDocs/Auto_Screenshot_Change_Docs_-_basic_components_-_variable0.png   "Scenario with variable declaration")
 
 
-As you can see in the `variable` configuration form below, Nussknacker inferred the data type of the `#input` variable from the information already available to Nussknacker.
+As you can see in the `variable` configuration form below, Nussknacker has inferred the data type of the `#input` variable. Nussknacker can do this based on the information available from the previous components.
 
 ![alt_text](../autoScreenshotChangeDocs/Auto_Screenshot_Change_Docs_-_basic_components_-_variable1.png "Variable declaration form")
 
 
-In the next example `#input` variable is used to create an expression returning a boolean value. If the input Kafka topic contains JSON objects and they contain `operation` field, the value of this field can be obtained in the following way:
+In the next example `#input` variable is used to create an expression returning a boolean value. If the input Kafka topic contains JSON objects and they contain an `operation` field, the value of the field can be obtained using the following pattern:
 
 
 `#input.operation`
 
-Note that internally Nussknacker converts JSON’s object into SpEL’s map.
-
-
 
 ![alt_text](../autoScreenshotChangeDocs/Auto_Screenshot_Change_Docs_-_basic_components_-_variable2.png "Screenshot_tooltip")
 
+Note that internally Nussknacker converts the JSON object into a SpEL map.
```

Comment:
We no longer use 'map' term - it is a[record](https://nussknacker.io/documentation/docs/scenarios_authoring/BasicNodes/#recordvariable) these days. 

Moreover, "internally converts" is debatable. I think Nu [maps scenario variables](https://nussknacker.io/documentation/docs/scenarios_authoring/Spel/#data-types-and-structures) to Java types; but I think this info is not critical here and can be omitted.

Reply:
I think internally converts is ok, we don't need to dabble in specifics here. 
Changed map to record.